lms/djangoapps/discussion/views.py

```diff
@@ -809,15 +807,11 @@ def css_dependencies(self):
         the files are loaded individually, but in production just the single bundle is loaded.
         """
         is_right_to_left = get_language_bidi()
-        if use_bootstrap_flag_enabled():
-            css_file = BOOTSTRAP_DISCUSSION_CSS_PATH
-            if is_right_to_left:
-                css_file = css_file.replace('.css', '-rtl.css')
-            return [css_file]
-        elif is_right_to_left:
-            return self.get_css_dependencies('style-discussion-main-rtl')
-        else:
-            return self.get_css_dependencies('style-discussion-main')
```

Comment:
Shoul we remove this from PIPELINE['STYLESHEETS']  in lms/envs/common.py ? @abutterworth

Reply:
Also the files that generate those files,

https://github.com/edx/edx-platform/blob/master/lms/static/sass/discussion/lms-discussion-main.scss and 
https://github.com/edx/edx-platform/blob/master/lms/static/sass/discussion/lms-discussion-main-rtl.scss

Reply:
Yes, I think we should delete https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L1834-L1845 and the files you mention.